app/src/main/res/layout/crf_step_layout_instruction.xml

```diff
@@ -120,6 +120,12 @@
             android:textAllCaps="false"
             android:onClick="goForwardClicked"/>
 
+        <org.researchstack.backbone.ui.views.SubmitBar
```

Comment:
This is a blocker, we don't use the SubmitBar in CRF and it shouldn't be added to our global instruction layout.  If you need a skip option for fitbit, add a simple underlined borderless button to the fitbit step layout.

Reply:
Why aren't we using SubmitBar? 

During testing I didn't notice a difference so I was actually planning to switch to code to use SubmitBar instead of a separate button so CrfInstructionStep could reuse the code from InstructionStep. It was confusing to see CrfInstructionStep extend InstructionStep but only support a subset of features (e.g. skip didn't work), since I think of subclasses being an extension/superset of features of the parent class.